src/MudBlazor/Styles/components/_navmenu.scss

```
@@ -157,7 +157,11 @@
     }
   }
 
-  &:focus:not(.mud-nav-link-disabled) {
+  &:focus:not(:focus-visible) {
+    background-color: transparent !important;
```

I verified this works without important and without assigning an arbitrary color. If it hadn't worked I would look at extending the targeting selector to avoid !important if possible.

```suggestion
    background-color: unset;
```

---

I agree to change value with unset.